proposals/0018-work-graphs.md

```diff
@@ -543,9 +549,12 @@ void GroupMemoryBarrier() { Barrier(GROUP_SHARED_MEMORY, GROUP_SCOPE); }
 void GroupMemoryBarrierWithGroupSync() {
   Barrier(GROUP_SHARED_MEMORY, GROUP_SCOPE | GROUP_SYNC);
 }
-
 ```
 
+> Note: These equivalent forms are not currently mapped to the previous DXIL
+> Barrier operation on prior shader models.  The new barrier is only available
+> on SM 6.8 and above, for now.
```

Comment:
I don't entirely know that this note's clarification is really required. We didn't provide qualifying statements like this to any of the other language additions in this proposal.

At a minimum we should keep the spec to what we've committed to and separate out future plans in a separate spec. Putting wording like "for now" or "yet" into a spec implies to a reader that this will change, and this change isn't on our roadmap.

```suggestion
> on SM 6.8 and above.
```